documentation/src/main/wiki/PerformanceTestsMethodology.md

```diff
@@ -0,0 +1,95 @@
+# Performance Tests Methodology
+
+## Introduction
+This page describes methodology of Knotx application performance tests.
+
+## Goals
+Goals of conducted tests are as following:
+* Assess system performance,
+* Assess system stability.
+
+### Metrics
+During our tests we focus on following metrics and correlate it to came out with observations:
```

Comment:
_During our tests, we focus on the following metrics which, when corellated, allow us to come up with concrete observations about the system:_